controllers/workspaceitem_controller.go

```diff
@@ -62,27 +63,30 @@ func (r *WorkspaceItemReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 	if instance.IsBeingDeleted() {
 		r.Log.Info(fmt.Sprintf("HandleFinalizer for %v", req.NamespacedName))
 		if err := r.handleFinalizer(instance); err != nil {
+			r.Recorder.Event(instance, corev1.EventTypeWarning, "deleting finalizer", fmt.Sprintf("Failed to delete finalizer: %s", err))
 			return ctrl.Result{}, fmt.Errorf("error when handling finalizer: %v", err)
 		}
-		r.Recorder.Event(instance, "Normal", "Deleted", "Object finalizer is deleted")
+		r.Recorder.Event(instance, corev1.EventTypeNormal, "Deleted finalizer", "Object finalizer is deleted")
 		return ctrl.Result{}, nil
 	}
 
 	if !instance.HasFinalizer(databricksv1alpha1.WorkspaceItemFinalizerName) {
 		r.Log.Info(fmt.Sprintf("AddFinalizer for %v", req.NamespacedName))
 		if err := r.addFinalizer(instance); err != nil {
+			r.Recorder.Event(instance, corev1.EventTypeWarning, "Adding finalizer", fmt.Sprintf("Failed to add finalizer: %s", err))
 			return ctrl.Result{}, fmt.Errorf("error when adding finalizer: %v", err)
 		}
-		r.Recorder.Event(instance, "Normal", "Added", "Object finalizer is added")
+		r.Recorder.Event(instance, corev1.EventTypeNormal, "Added finalizer", "Object finalizer is added")
```

Comment:
Could you please change it to `r.Recorder.Event(instance, corev1.EventTypeNormal, "Added", "Object finalizer is added")`?